src/main/resources/templates/place/page.html

```diff
@@ -100,16 +99,20 @@ <h2>Rating criteria</h2>
 <script th:inline="javascript">
     /*<![CDATA[*/
 
+<script th:inline="javascript" src="/scripts/rating.js"></script>

```

Comment:
Broken whole <script> section after merge, should be probably:

```
<script th:replace="~{fragments :: placeMap-script}"></script>
<script th:inline="javascript">
    /*<![CDATA[*/

    let latitude = [[${place.latitude}]];
    let longitude = [[${place.longitude}]];
    let name = [[${place.name}]];
    let rating = [[${place.getAverageRating()}]];

    /*]]>*/
</script>
<script src="/scripts/placeMap.js"></script>
<script src="/scripts/imageFetcher.js"></script>
<script th:if="${usersRatings}" src="/scripts/rating.js"></script>
<script th:if="${usersRatings}" type="module" src="/scripts/imageUpload.js"></script>
```

Reply:
fixed